File: Bidding/Models/ViewModels/Auctions/Details/AboutAuctionDetailsModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace Bidding.Models.ViewModels.Auctions.Details
{
    public class AboutAuctionDetailsModel
    {
        public string AuctionName { get; set; }

        [Column(TypeName = "decimal(18,2)")]
        public decimal AuctionStartingPrice { get; set; }
        public string AuctionFormat { get; set; }
        public DateTime? AuctionStartDate { get; set; }
        public DateTime AuctionApplyTillDate { get; set; }
        public DateTime AuctionEndDate { get; set; }
        public string ItemEvaluation { get; set; }
        public List<string> AuctionImageUrls { get; set; }
        public List<string> AuctionDocumentUrls { get; set; }     
    }
}

File: Bidding/ClientApp/src/app/auctions/models/details/about-auction-details.model.ts
import { AuctionDocumentModel } from "./auction-document.model";

export interface AboutAuctionDetailsModel {
  auctionName: string;
  auctionStartingPrice: number;
  auctionValueAddedTax: string;
  auctionFormat: string;
  auctionExternalWebsite: string;
  auctionStartDate: Date;
  auctionApplyTillDate: Date;
  auctionEndDate: Date;
  itemEvaluation: string;
  auctionImageUrls: string[];
  auctionDocumentUrls: AuctionDocumentModel[];
}
